diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -313,6 +314,23 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.WRONG_NEWLINES)
+    fun `newline should be placed only after colon`() {
+        lintMethod(
+            """
+                    |fun foo(a
+                    |        : Int,
+                    |        b
+                    |        : Int) {
+                    |    bar(a, b)
```

Comment:
How will some messy code like
```kotlin
fun foo(a
    : Int, b:
    Int, c: Int)
```
be fixed now? With respect to number of arguments on the same line, long line and newlines after `:`?